conf/application.conf

```diff
@@ -16,7 +16,7 @@ include "frontend.conf"
 
 
 # Fix for SSLEngine closed already exception
-http-verbs.retries.ssl-engine-closed-already.enabled = true
+http-verbs.retries.ssl-engine-closed-already.enabled = false
```

Comment:
Shouldn't the retry mechanism stay enabled?

Reply:
Forgot about that - I was tinkering on Friday